daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/EXIInfosetInputter.scala

```diff
@@ -28,15 +28,15 @@ import java.io.ByteArrayInputStream
 import java.io.ByteArrayOutputStream
 import java.io.BufferedOutputStream
 import javax.xml.transform.stream.StreamResult
-import org.apache.daffodil.util.MaybeBoolean
+import org.apache.daffodil.util.{ MaybeBoolean, Misc }
 import org.apache.daffodil.dpath.NodeInfo
 
 object EXIInfosetInputter {
-  def ConvertEXIToXMLWithExificient(input: java.io.InputStream): java.io.InputStream = {
-    val xsdLocation = "daffodil-lib/src/main/resources/org/apache/daffodil/xsd/XMLSchema.xsd"
+  def ConvertEXIToXML(input: java.io.InputStream): java.io.InputStream = {
+    val xsdLocation = Misc.getRequiredResource("org/apache/daffodil/xsd/XMLSchema.xsd")
```

Comment:
I suppose I should have prefaced this commit by saying that this was mainly getting the existing code working.  I had some of the same thoughts but since it was working I thought it might be worth a review.